onboard/import.go

```diff
@@ -58,17 +61,20 @@ func CreateImporter(ctx context.Context, logger logging.Logger, config *Config)
 		repository:         config.Repository,
 		inventoryGenerator: config.InventoryGenerator,
 		logger:             logger,
+		rocks:              config.Rocks,
 		CatalogActions:     config.CatalogActions,
 	}
+
 	if res.CatalogActions == nil {
-		res.CatalogActions = NewCatalogActions(config.Cataloger, config.Repository, config.CommitUsername, logger)
+		res.CatalogActions = buildRepoActions(config, logger)
 	}
+
 	previousCommit, err := res.CatalogActions.GetPreviousCommit(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get previous commit: %w", err)
 	}
 	res.previousCommit = previousCommit
-	shouldSort := res.previousCommit != nil
+	shouldSort := res.previousCommit != nil || res.rocks
```

Comment:
why is this removed? don't we need the inventory to return sorted?

Reply:
What is removed? For rocks you always have to return the inventory in order.